sdk/attestation/attestation/test/public/attestationTests.spec.ts

```diff
@@ -197,15 +197,15 @@ describe("[AAD] Attestation Client", function () {
     const binaryRuntimeData = base64url.decodeString(_runtimeData);
     const client = createRecordedClient(recorder, endpointType);
 
-    {
-      // You can't specify both runtimeData and runtimeJson.
-      await expect(
```

Comment:
expect wasn't capturing somehow.
Maybe a breaking change or something, hence updated to assert instead.